src/rules/__tests__/use-next-tooltip.test.js

```diff
@@ -0,0 +1,32 @@
+const rule = require('../use-next-tooltip')
+const {RuleTester} = require('eslint')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+})
+
+ruleTester.run('use-next-tooltip', rule, {
+  valid: [
+    `import {Tooltip} from '@primer/react/next'`,
+    `import {UnderlineNav, Button} from '@primer/react';
```

Comment:
Not sure if we need this line of code too?

Reply:
Probably not 😆 I was just adding examples to replicate real life cases but if it is confusing or you think it doesn't seem very related, I can totally remove it.